### ISSUE TITLE
Missing sqlite file

### ISSUE THREAD

**Issue by seresistvanandras** [etherclust/etherclust#1]
Can you please upload the sqlite file? config.py references config.dbFile="data/output/tempdb.sqlite" which is missing currently from the data folder. Or are we supposed to generate it from the given csv files? Could you please elaborate on this in the Readme? Thanks in advance!

**Comment by friedhelmvictor** [comment]
Yes, the tempdb.sqlite is supposed to be generated from the `transactionsFile` specified in the config. It should contain Ethereum transactions, not token transfers.
The db is used to store a list of active accounts. In other words, EOA addresses that have at some point in time sent a regular transaction.
Depending on your timeframe of analysis, this database may contain more or less addresses. 

The number of such accounts is rather large, about 50 million, so I can't easily upload it. I will see if I can find a way, but can't make any promises.
On the other hand, there are about 20 million accounts that have never been active.
One such account is `0x0000000000000000000000000000000000000000`, but there are many more.
As people use such addresses to "burn" tokens, the airdrop collection heuristic should avoid capturing patterns where airdropped tokens are sent to a such a throwaway address.

There could be other ways to find the list of active addresses.
With a --fat-db parity client, we can also access a list of addresses.
https://wiki.parity.io/JSONRPC-parity-module#parity_listaccounts
Then you would check the nonce of each account, which tells you if it has been active.
But I think this is also not very practical.

I will update the readme soon.
